gossip/c_llr_process_votes_test.go

```diff
@@ -129,6 +158,30 @@ func (s *LLRCallbacksTestSuite) Test_processBlockVoteLLRVotingDoubleSignIsMet()
 
 	*/
 
+	bs, es := s.env.store.GetHistoryBlockEpochState(1)
+	s.Require().NotNil(es)
+	s.Require().NotNil(bs)
+
+	var bv1 inter.LlrSignedBlockVotes
+	bv1 = s.bvs
+	
+	bv1.Val.Start = bs.LastBlock.Idx - 1
+	bv1.Val.Epoch = 1
+	bv1.Val.Votes = []hash.Hash{hash.Zero,hash.HexToHash("0x01"), hash.HexToHash("0x02")}
+
+	em := s.env.emitters[1] 
+	e, err := em.EmitEvent()
+	s.Require().NoError(err)
+	bv2 := inter.AsSignedBlockVotes(mutableEventPayloadFromImmutable(e))
+
+	bv2.Val.Start = bs.LastBlock.Idx
+	bv2.Val.Epoch = 1
+	bv2.Val.Votes = []hash.Hash{hash.HexToHash("0x02"), hash.HexToHash("0x03")}
+
+
+	s.Require().NoError(s.env.ProcessBlockVotes(bv1))
```

Comment:
Any ideas how can I return errLLRVotingDoubleSign on L182 or L183?

Reply:
@uprendis @rus-alex I would appreciate some guidance regarding calling this test by set of validators.